static/css/components/header-bar.less

```diff
@@ -145,6 +145,15 @@
       height: 44px;
       padding: 0 3px;
     }
+
+
+    #toggle {
```

Comment:
you'll want to use something less generic than #toggle here and I'd recommend using a class selector rather than ID. 

I suggest using:  hamburger-component__checkbox.